commons/src/main/java/org/infinispan/commons/util/CollectionFactory.java

```diff
@@ -101,8 +103,7 @@ public static int computeCapacity(int expectedSize, float loadFactor) {
    }
 
    public static <K, V> ConcurrentMap<K, V> makeBoundedConcurrentMap(int maxSize) {
-      return new BoundedEquivalentConcurrentHashMapV8<>(maxSize,
-            AnyEquivalence.getInstance(), AnyEquivalence.getInstance());
+      return new CaffeineConcurrentMap<>(Caffeine.newBuilder().maximumSize(maxSize).build());
```

Comment:
I have another PR for that, already has 27K deletions, didn't want to make to completely overload this one.  I will be submitting it to Preview today.